hack/jenkins/e2e.sh

```diff
@@ -180,12 +176,6 @@ GCE_PARALLEL_FLAKY_TESTS=(
     "Namespaces.*should\sdelete\sfast"
     "PD"
     "ServiceAccounts"
-    "Services.*change\sthe\stype"
-    "Services.*functioning\sexternal\sload\sbalancer"
-    "Services.*identically\snamed"
-    "Services.*release.*load\sbalancer"
-    "Services.*endpoint"
```

Comment:
`Services should serve multiport endpoints from pods` flaked in https://goto.google.com/k8s-test/view/Flaky/job/kubernetes-e2e-gce-parallel-flaky/7477/ (though so did lots of tests...)